checker/jdk/index/src/java/nio/CharBuffer.java

```diff
@@ -0,0 +1,1524 @@
+/*
+ * Copyright (c) 2000, 2013, Oracle and/or its affiliates. All rights reserved.
+ * ORACLE PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ */
+
+// -- This file was mechanically generated: Do not edit! -- //
```

Comment:
We should probably remove this notice, since we are obviously editing it and there's no reason others shouldn't, either.

Reply:
We want to keep the diffs between the annotated JDK and the JDK as small as possible, so this comment should remain.